web/themes/custom/badcamp2017/css/styles.css

```diff
@@ -5241,6 +5241,18 @@ a.thumbnail image {
   clear: both;
 }
 
+a {
+  color: #0071bd;
+}
+
+a:hover {
+  color: #24a7ff;
+}
+
+.pagination {
+  text-align: center;
+}
+
 #block-badcamp2017-branding {
```

Comment:
not a fan of using this ID because a) it's an ID (classes are better for CSS-- IDs are generally reserved for JS) but more importantly b) we plan to re-use this site's theme, so 2017 will be outdated soon.